support-frontend/assets/pages/paper-subscription-checkout/components/postcodeFinderReducer.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import { type Dispatch } from 'redux';
+import { type Option } from 'helpers/types/option';
+
+export type Address = {|
+  lineOne?: string,
+  lineTwo?: string,
+  city?: string,
```

Comment:
Think you might need `county`/`state` in here as well?

Reply:
Actually you don't do you, ignore this.

Reply:
haha nope, but if we need more we can always add to it!